packages/next/client/app-index.tsx

```diff
@@ -113,6 +114,12 @@ if (document.readyState === 'loading') {
   DOMContentLoaded()
 }
 
+// Skip certain custom errors which are not expected to throw on client
+function onRecoverableError(err: any) {
+  if (err.digest === NEXT_DYNAMIC_NO_SSR_CODE) return
+  throw err
```

Comment:
What will happen when this case is hit? Will it mess with the stacktrace?

Reply:
we shouldn't throw here, will change to the default one in react, it only reports or logs the error